configure.py

```diff
@@ -1540,6 +1540,7 @@ def get_warning_options(cxx):
         '-Wno-mismatched-tags',  # clang-only
         '-Wno-c++11-narrowing',
         '-Wno-overloaded-virtual',
+        '-Wno-unused-command-line-argument',
```

Comment:
need morivation for this patch

Cc @tchaikov

Reply:
IIRC, you identified that it was an issue in distcc, right? See https://github.com/distcc/distcc/issues/504

Reply:
But I think it's more like a misconfiguration of the CPP option of either ccache or distcc in your environment.